network/edge-services/reference-content/understanding-pricing.mdx

```diff
@@ -0,0 +1,169 @@
+---
+meta:
+  title: Understanding Edge Services pricing
+  description: Find answers to all your questions about how pricing for Edge Services works, how to calculate your billing, and the different subscription plans available. Different scenarios are presented, with explanations of how you would be billed in each one.
+content:
+  h1: Understanding Edge Services pricing
+  paragraph: Find answers to all your questions about how pricing for Edge Services works, how to calculate your billing, and the different subscription plans available. Different scenarios are presented, with explanations of how you would be billed in each one
+tags: edge-services general-availability pricing billing subscription-plan pro-rata
+dates:
+  validation: 2024-10-16
+  creation: 2024-10-16
+categories: 
+  - network
+---
+
+Find out everything you need to know about how Edge Services is billed in this document.
+
+## Edge Services billing basics
+
+Edge Services billing is based on [monthly subscription plans](https://www.scaleway.com/en/pricing/network/#edge-services).Three plans are available, with different prices and consumption limits:
+
+- **Starter**
+- **Professional**
+- **Advanced** 
+
+When you subscribe to a plan, you are billed its flat monthly fee, which allows you to:
+
+- Run a fixed maximum number of Edge Services [pipelines](/network/edge-services/concepts/#pipeline) for the month. They can be for Object Storage or Load Balancer origins, or a mixture of both.
+- Egress a fixed maximum amount of data from all your pipelines' [caches](/network/edge-services/concepts/#cache).
+
+If you subscribe to a plan, and exceed its monthly limits for pipelines or cache data, you will incur additional charges that month. 
+
+Essentially, your Edge Services monthly bill is made up of your **monthly subscription plan price** + **any additional pipeline charges incurred** + **any additional cache charges incurred**.
+
+For full details of the price and limits of each plan, refer to the [pricing page](https://www.scaleway.com/en/pricing/network/#edge-services).
+
+## Keeping track of your Edge Services consumption
+
+We provide a number of ways to keep track of your Edge Services consumption:
+
+- View, in relation to your plan's limit, how many pipelines you currently have and how much cache data you have consumed so far this month, in the **Pipelines** tab of your [Edge Services dashboard](https://console.scaleway.com/edge-services)
```

Comment:
We won't display this information at first (but it will be available quickly after the GA)